services/placement/types.go

```diff
@@ -36,13 +36,39 @@ type Algorithm interface {
 	// RemoveInstances removes a list of instances from the placement.
 	RemoveInstances(p services.Placement, leavingInstanceIDs []string) (services.Placement, error)
 
-	// ReplaceInstance replace a instance with new instances.
-	ReplaceInstance(p services.Placement, leavingInstanceID string, addingInstances []services.PlacementInstance) (services.Placement, error)
+	// ReplaceInstance replace a list of instances with new instances.
+	ReplaceInstances(
```

Comment:
Yeah... I could do that, I was trying to take the chance to keep less code, if you think the effort of fixing the broken dependent is too much I'm ok to keep the old interface.